# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clip/workaround_nolibgalde_case.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

cliplibs will fail if clip-glade2 doesn't find libglade2,
this workaround is very very ugly but 'acceptable' in this specific case
//mnemoc

--- ./cliplibs/Makefile.orig	2006-03-08 08:22:08.000000000 -0300
+++ ./cliplibs/Makefile	2006-03-08 08:23:03.000000000 -0300
@@ -11,7 +11,7 @@
 	do \
 		(cd $$dir ;\
 		[ -x ./configure ] && ./configure ;\
-		$(MAKE) $@ );\
+		$(MAKE) $@ || true );\
 	done
 
 clean:
